README.md

```diff
@@ -1,4 +1,4 @@
-# Fishbrain Engineering Blog
+# [Fishbrain Engineering Blog](https://engineering.fishbrain.com/)
```

Comment:
Not sure I like the idea of combining link and header. Can we keep the title and add link below?

Reply:
Sure!